services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -118,7 +119,7 @@ boolean someBrokerWithoutLeader() {
      * @return -1 if partitions do not all have the same replication factor possibly indicating that the topic is
      * undergoing maintenance
      */
-    int replicationFactor() {
+    int currentReplicationFactor() {
```

Comment:
I am not sure if we should make this `currentReplicationFactor`. By definition any state of `TopicState` should be current, right?